src/components/themed/SelectFioAddress.js

```diff
@@ -52,20 +55,27 @@ class SelectFioAddressComponent extends React.PureComponent<Props, LocalState> {
     super(props)
     this.state = {
       loading: false,
+      bundlesUpdated: false,
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name